test/integration/test_container_resolvers.py

```diff
@@ -480,6 +480,11 @@ class MulledTestCase:
     mulled_hash = "mulled-v2-8186960447c5cb2faa697666dc1e6d919ad23f3e:a6419f25efff953fc505dbd5ee734856180bb619-0"
 
 
+class MulledTestCaseWithBuildInfo:
+    tool_id = "mulled_example_multi_2"
```

Comment:
How does this work if you don't add the tool to the sample tool config file ?

Reply:
Good point. I forgot to add an actual test case.